Corona-Warn-App/src/main/java/de/rki/coronawarnapp/risk/TimeVariables.kt

```diff
@@ -34,7 +34,7 @@ object TimeVariables {
      * The maximal runtime of a transaction
      * In milliseconds
      */
-    private const val TRANSACTION_TIMEOUT = 180000L
+    private const val TRANSACTION_TIMEOUT = 8 * 60 * 1000L
```

Comment:
Maybe also add the background task system timeout as a comment here so in future it wont be set higher than 10 minutes by accident?